apps/sriov-kernel-nse/ponger-configmap.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ponger
+data:
+  pong.sh: |
+    #!/bin/bash +x
+    # shellcheck disable=SC2064
+
+    link="$1"
+    ip_addr="$2"
+
+    # Don't forget to delete IP address on exit
+    trap "ip addr del dev '${link}' '${ip_addr}'" err exit
+
+    # Add IP address and wake up the link
+    ip addr add dev "${link}" "${ip_addr}" || exit 1
+    ip link set dev "${link}" up || exit 2
+
+    # Wait for exit
+    sleep 60m
```

Comment:
I think this can be used via `kubectl exec`

Reply:
It is need to be executed as a main process for the ponger pod so it is better to be started from the .yaml config.